packages/manager/src/features/Linodes/LinodeCreatev2/Tabs/Marketplace/utilities.ts

```diff
@@ -132,9 +132,50 @@ const getDoesStackScriptMatchQuery = (
  * @param stackscript The StackScript to compare aginst
  * @returns true if the given StackScript has the given category
```

Comment:
```suggestion
 * @returns true if the given app has the given category
```
Optional: for consistency